### PR TITLE
Update feature_bluetooth.md of Japanese document.

### PR DESCRIPTION
**Pull request by shelaf**
## Description

Update Japanese document.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_bluetooth.md [hunk 1]
```diff
@@ -30,7 +30,10 @@ Bluefruit UART friend は SPI friend に変換することができますが、
 
 <!-- FIXME: Document bluetooth support more completely. -->
 ## Bluetooth の Rules.mk オプション
-これらのうちの1つだけを使ってください
+
+現在サポートされている Bluetooth チップセットは [N-キーロールオーバー (NKRO)](ja/reference_glossary.md#n-key-rollover-nkro) をサポートしていません。そのあめ、`rules.mk` に `NKRO_ENABLE = no` が含まれていなければなりません。
```
Comment (s-show) [inline docs/ja/feature_bluetooth.md:34]: ユーザーに対する指示なので、「〜するように」という形で示す方が良いかなと思います。
```suggestion
現在サポートされている Bluetooth チップセットは [N-キーロールオーバー (NKRO)](ja/reference_glossary.md#n-key-rollover-nkro) をサポートしていません。そのため、`rules.mk` に `NKRO_ENABLE = no` を含めなければなりません。
```

**Review by s-show — APPROVED** [review]

File: docs/ja/feature_bluetooth.md [hunk 2]
```diff
@@ -30,7 +30,10 @@ Bluefruit UART friend は SPI friend に変換することができますが、
 
 <!-- FIXME: Document bluetooth support more completely. -->
 ## Bluetooth の Rules.mk オプション
-これらのうちの1つだけを使ってください
+
+現在サポートされている Bluetooth チップセットは [N-キーロールオーバー (NKRO)](ja/reference_glossary.md#n-key-rollover-nkro) をサポートしていません。そのため、`rules.mk` に `NKRO_ENABLE = no` を含めなければなりません。
+
+Bluetooth を有効にするには、これらのうちの1つだけを使ってください:
```
Comment (mtei) [inline docs/ja/feature_bluetooth.md:36]: these は、この場合には、「以下の」と訳すのが適切です。

```suggestion
Bluetooth を有効にするには、以下のうちの1つだけを使ってください:
```

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**